.yarn/versions/94f0ddf0.yml

```
@@ -0,0 +1,7 @@
+releases:
+  "@yarnpkg/plugin-essentials": patch
```

```suggestion
  "@yarnpkg/plugin-essentials": minor
```

Also need to mark `@yarnpkg/cli` for release.